runner.py

```diff
@@ -93,6 +102,28 @@ def get_configs(path: Path) -> List[str]:
         for params_set in config['cases']:
             params = common_params.copy()
             params.update(params_set.copy())
+
+            device = []
+            if 'device' not in params:
+                if 'sklearn' in params['lib']:
+                    logging.info('The device parameter value is not defined in config, '
+                                 'none is used')
```

Comment:
Move up from `if` because `--device` might be added later for xgboost, for example.

Reply:
For now other configs not support device parameter and such message may be confusing